### PR TITLE
Footer icons and horizontal scrollbar fix

### LINKED ISSUE #62: Fix footer and horizontal scrollbar

### PR DESCRIPTION
**Pull request by Lunchii**
Closes #62 
## Overview
The icons were not hoverable nor clickable (changed the z-index).
On some devices, the vertical scrollbar caused the horizontal scrollbar to appear (hidden overflow-x)

### CODE REVIEW

**Review by jerpyk — COMMENTED** [review]
icon shows on top of mobile nav bar
Screenshot 2024-12-29 at 8 19 43 PM

**Review by ZestyScrubz — DISMISSED** [review]
👍

File: components/footer/Footer.tsx [hunk 1]
```diff
@@ -8,13 +8,13 @@ const Footer = () => {
     <div>
       <div className="bg-[#7a7c7b] w-full h-[50px] flex items-center justify-center">
         <div className="flex gap-4">
```
Comment (jerpyk) [inline components/footer/Footer.tsx:10]: i think it's here for fixing the footer icons on top of mobile nav bar

**Review by jerpyk — CHANGES_REQUESTED** [review]

File: components/footer/Footer.tsx (hunk 1 above)
Comment (Lunchii) [reply components/footer/Footer.tsx:10]: its because i made the z-index 50 because it would work with anything else lower so thats probably why

**Pull request merged**